src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    return (y) => {
+      count++;
+
+      if (count >= 3 && !(count % 2)) {
```

Comment:
Wow, didn't notice at first, it's unclear what the second check is
```suggestion
      if (count >= 3 && count % 2 === 0) {
```